ci/baremetal/baremetal-cluster.lokocfg.envsubst

```diff
@@ -42,6 +42,18 @@ cluster "bare-metal" {
   }
```

Comment:
Using test 2 times consecutively in commit heading sounds a little vague. Seeing the code it should be regarding CI changes. Can you please rephrase it?